engine/src/main/java/org/terasology/world/block/family/AbstractBlockFamily.java

```diff
@@ -29,13 +32,23 @@
     private BlockUri uri;
     private Set<String> categories = Sets.newHashSet();
```

Comment:
This does not make much sense in the scope of families. these are already provided by the block itself.